mabl snippets/GetDateComponents.js

```diff
@@ -0,0 +1,184 @@
+/**
+Returns an object containing values assosiated todays date.
+IE compatiable: true
```

Comment:
Nit: spelling of associated, today's, compatible.

Reply:
Done and thank you!